src/browser/components/Visualization/components/Button/Button.css

```diff
@@ -0,0 +1,18 @@
+button {
+  background-color: #66757f;
+  border: none;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-variant-caps: all-petite-caps;
+  font-size: 100%;
+  padding: 0.5rem 2rem;
+  text-align: center;
+  display: inline-block;
+  margin: 0px 2px;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #778894;
+  color: white;
```

Comment:
is this really needed?

Reply:
Shouldn't be, I just copied this. Will remove.